Libraries/WebSocket/RCTWebSocketModule.m

```diff
@@ -48,6 +48,19 @@ - (void)dealloc
 RCT_EXPORT_METHOD(connect:(NSURL *)URL protocols:(NSArray *)protocols headers:(NSDictionary *)headers socketID:(nonnull NSNumber *)socketID)
 {
   NSMutableURLRequest *request = [NSMutableURLRequest requestWithURL:URL];
+  
+  // We load cookies from sharedHTTPCookieStorage (shared with XHR and
+  // fetch). To get HTTPS-only cookies for wss URLs, replace wss with https
+  // in the URL.
+  NSURLComponents *components = [NSURLComponents componentsWithURL:URL resolvingAgainstBaseURL:true];
+  if ([components.scheme isEqualToString:@"wss"]) {
```

Comment:
According to RFC 3986, it should be lowercase however I will verify and change if needed. Apple doc doesn't say anything.